tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Modify from single- to mixed-dimensional poroelasticity."""
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>